modules/ptp-dual-nics.adoc

```diff
@@ -6,38 +6,47 @@
 [id="ptp-dual-nics_{context}"]
 = Using dual-NIC Intel E810 hardware with PTP
```

Comment:
Because this is a concept topic, rewrite the title to avoid using a gerund. For example, "Dual-NIC Intel E810 hardware with PTP".

Also, elsewhere in this PR, I noticed you replaced "dual" with "2". Should that be done here as well?